### PR TITLE
Resolve nanopb already exists problem when brew with "--HEAD"

### PR DESCRIPTION
**Pull request by muxi**
Resolves Issue #90

### CODE REVIEW

File: Formula/grpc.rb [hunk 1]
```diff
@@ -22,7 +22,7 @@ def install
     # UGLY HACK: we depend on headers from third_party/nanopb being available
     # so we just fetch them from github ourselves.
     # TODO(jtattermusch): find a better solution
```
Comment (jtattermusch) [inline Formula/grpc.rb]: please update the comment to reflect what you are doing and why.

**Review by jtattermusch — APPROVED** [review]
LGTM after updating the comment and manually testing that the new formula works well.

File: Formula/grpc.rb (hunk 1 above)
Comment (muxi) [reply Formula/grpc.rb]: roger

**Comment by muxi** [comment]
Instead of cloning nanopb, how about a "git submodule update --init" here? That seems to resolve this problem and some potential future problem in a graceful way.

**Pull request merged**